src/engine/Traits/EulerMovement.ts

```diff
@@ -8,9 +8,8 @@ module ex.Traits {
          var seconds = delta / 1000;
          
          var totalAcc = actor.acc.clone();
-         // Fixed actors are not affected by global acceleration
-         if (actor.collisionType !== ex.CollisionType.Fixed &&
-            actor.collisionType !== ex.CollisionType.PreventCollision &&
+         // Only active vanilla actors are affected by global acceleration
+         if (actor.collisionType === ex.CollisionType.Active &&
             !(actor instanceof UIActor) &&
```

Comment:
Is there a better way to differentiate types that shouldn't be affected? It would be nice to have something more forward thinking.